definitions/divorce/json/CaseEvent/CaseEvent-share-a-case-prod.json

```diff
@@ -55,68 +55,6 @@
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
   },
-  {
```

Comment:
All events are already defined in the core file. They are pretty much the same. The only difference is `preConstionState(s)` - events defined in the core file have more states, but they always have all pre states that we used to have here.